nav2_costmap_2d/costmap_plugins.xml

```diff
@@ -1,17 +1,21 @@
 <class_libraries>
-  <library path="liblayers">
+  <library path="layers">
```

Comment:
Unless pluginlib has changed in ROS2, I'm 99% sure you need that lib there so the plugins register correctly.

Reply:
I believe it threw warnings with the lib prefix, but I can verify that the plugins register correctly.

Reply:
the add_library in ROS1 would add a `lib` prefix on top, so you needed the lib to whatever the linkable name you gave in the CMakeLists to match. It was a little subtle (and annoying) necessity.